crates/languages/src/python/runnables.scm

```diff
@@ -82,3 +82,19 @@
         )
     )
 )
+
+; module main method
+(
+    (module
+        (if_statement
+            condition: (comparison_operator
+                (identifier) @run @_lhs
+                operators: "=="
+                (string) @_rhs
+            )
+            (#eq? @_lhs "__name__")
+            (#match? @_rhs "^[\"']__main__[\"']$")
```

Comment:
This will now match both `'__main__'` and `"__main__"` literals. I originally wanted to capture the first quote and assert that the second quote be equal, but `"^([\"'])__main__\1$"` did not work.

This isn't a problem, because  `'__main__"` and `"__main__'` aren't even valid Python string literals in the first place, so such matches cannot occur.